src/test/java/br/com/caelum/carangobom/CarangoBomApplicationTests.java

```diff
@@ -0,0 +1,84 @@
+package br.com.caelum.carangobom;
+
+import br.com.caelum.carangobom.controller.BrandController;
+import br.com.caelum.carangobom.domain.Brand;
+import br.com.caelum.carangobom.repository.BrandRepository;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
+
+@SpringBootTest
+class CarangoBomApplicationTests {
+
+    @InjectMocks
+    private BrandController brandController;
```

Comment:
I think it's better we test the 'brand' side of business in its own files: brandControllerTests, brandRepositoryTests and so on